### PR TITLE
feat(EMS-1817-1819): Policy and exports - Change from 499,999 to 500,000

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made:
* Changed policy and exports single and multiple policy from 499,999 to 500,000
* Updated maximums and error messages
* Updated tests
* Husky lint changes

### CODE REVIEW

**Review by ttbarnes — COMMENTED** [review]
Do you think we should add some E2E test coverage for submitting with the maximum amount? Something like this

```js
application-submission/submit-an-application-single-policy-type-maximum-cover.spec.js
application-submission/submit-an-application-multiple-policy-type-maximum-cover.spec.js
```

We need to tidy up the `application-submission` E2E directory

File: e2e-tests/cypress/e2e/journeys/insurance/application-submission/submit-an-application-multiple-policy-type-maximum-cover.spec.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  });
+
+  after(() => {
+    cy.deleteApplication(referenceNumber);
+  });
+
+  it(`should redirect to ${APPLICATION_SUBMITTED}`, () => {
+    const expectedUrl = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${APPLICATION_SUBMITTED}`;
+
+    cy.url().should('eq', expectedUrl);
```
Comment (ttbarnes) [inline e2e-tests/cypress/e2e/journeys/insurance/application-submission/submit-an-application-multiple-policy-type-maximum-cover.spec.js:37]: `cy.assertUrl()`

File: e2e-tests/cypress/support/insurance/complete-and-submit-multiple-contract-policy-form.js [hunk 2]
```diff
@@ -21,14 +21,19 @@ const {
...
 
   multipleContractPolicyPage[TOTAL_MONTHS_OF_COVER].input().select(application.POLICY_AND_EXPORTS[TOTAL_MONTHS_OF_COVER]);
   cy.keyboardInput(multipleContractPolicyPage[TOTAL_SALES_TO_BUYER].input(), application.POLICY_AND_EXPORTS[TOTAL_SALES_TO_BUYER]);
-  cy.keyboardInput(multipleContractPolicyPage[MAXIMUM_BUYER_WILL_OWE].input(), application.POLICY_AND_EXPORTS[MAXIMUM_BUYER_WILL_OWE]);
+
+  if (policyAndExportsMaximumValue) {
+    cy.keyboardInput(multipleContractPolicyPage[MAXIMUM_BUYER_WILL_OWE].input(), APPLICATION.POLICY_AND_EXPORT.MAXIMUM_BUYER_CAN_OWE);
+  } else {
+    cy.keyboardInput(multipleContractPolicyPage[MAXIMUM_BUYER_WILL_OWE].input(), application.POLICY_AND_EXPORTS[MAXIMUM_BUYER_WILL_OWE]);
+  }
```
Comment (ttbarnes) [inline e2e-tests/cypress/support/insurance/complete-and-submit-multiple-contract-policy-form.js:36]: The conditions do the same thing 😛

File: e2e-tests/cypress/support/insurance/complete-and-submit-multiple-contract-policy-form.js [hunk 3]
```diff
@@ -21,14 +21,19 @@ const {
   },
 } = FIELD_IDS;
 
-export default () => {
+export default ({ policyAndExportsMaximumValue }) => {
```
Comment (ttbarnes) [inline e2e-tests/cypress/support/insurance/complete-and-submit-multiple-contract-policy-form.js:24]: documentation?

**Review by ttbarnes — APPROVED** [review]
Great work

**Review by ttbarnes — APPROVED** [review]
